code/datums/customization_styles.dm

```diff
@@ -0,0 +1,593 @@
+				id = "minipig"
+				fem = 1
+				masc = 1
+			pig
+				name = "Pigtails"
+				id = "pig"
+				fem = 1
+			ponytail
+				name = "Ponytail"
+				id = "ponytail"
+				fem = 1
+				masc = 1
+			geisha_s
+				name = "Shimada"
+				id = "geisha_s"
+				fem = 1
+			twotail
+				name = "Split-Tails"
+				id = "twotail"
+				masc = 1
+			wavy_tail
+				name = "Wavy Ponytail"
+				id = "wavy_tail"
+				fem = 1
+		gimmick
+			afroHA
+				name = "Afro: Alternating Halves"
+				id = "afroHA"
+			afroRB
+				name = "Afro: Rainbow"
+				id = "afroRB"
+			bart
+				name = "Bart"
+				id = "bart"
+			ewave_s
+				name = "Elegant Wave"
+				id = "ewave_s"
+			flames
+				name = "Flame Hair"
+				id = "flames"
+			goku
+				name = "Goku"
+				id = "goku"
+			homer
+				name = "Homer"
+				id = "homer"
+			jetson
+				name = "Jetson"
+				id = "jetson"
+			sailor_moon
+				name = "Sailor Moon"
+				id = "sailor_moon"
+			sakura
+				name = "Sakura"
+				id = "sakura"
+			wiz
+				name = "Wizard"
+				id = "wiz"
+			xcom
+				name = "X-COM Rookie"
+				id = "xcom"
+			zapped
+				name = "Zapped"
+				id = "zapped"
+	moustache
+		fu
+			name = "Biker"
+			id = "fu"
+		chaplin
+			name = "Chaplin"
+			id = "chaplin"
+		dali
+			name = "Dali"
+			id = "dali"
+		hogan
+			name = "Hogan"
+			id = "hogan"
+		devil
+			name = "Old Nick"
+			id = "devil"
+		robo
+			name = "Robotnik"
+			id = "robo"
+		selleck
+			name = "Selleck"
+			id = "selleck"
+		villain
+			name = "Twirly"
+			id = "villain"
+		vandyke
+			name = "Van Dyke"
+			id = "vandyke"
+		watson
+			name = "Watson"
+			id = "watson"
+	beard
+		abe
+			name = "Abe"
+			id = "abe"
+		bstreak
+			name = "Beard Streaks"
+			id = "bstreak"
+		braided
+			name = "Braided Beard"
+			id = "braided"
+		chin
+			name = "Chinstrap"
+			id = "chin"
+		fullbeard
+			name = "Full Beard"
+			id = "fullbeard"
+		gt
+			name = "Goatee"
+			id = "gt"
+		hip
+			name = "Hipster"
+			id = "hip"
+		longbeard
+			name = "Long Beard"
+			id = "longbeard"
+		longbeardfade
+			name = "Long Beard: Faded"
+			id = "longbeardfade"
+		motley
+			name = "Motley"
+			id = "motley"
+		neckbeard
+			name = "Neckbeard"
+			id = "neckbeard"
+		puffbeard
+			name = "Puffy Beard"
+			id = "puffbeard"
+		tramp
+			name = "Tramp"
+			id = "tramp"
+		trampstains
+			name = "Tramp: Beard Stains"
+			id = "trampstains"
+	sideburns
+		elvis
+			name = "Elvis"
+			id = "elvis"
+	eyebrows
+		eyebrows
+			name = "Eyebrows"
+			id = "eyebrows"
+		thufir
+			name = "Huge Eyebrows"
+			id  = "thufir"
+	makeup
+		eyeshadow
+			name = "Eyeshadow"
+			id = "eyeshadow"
+		lipstick
+			name = "Lipstick"
+			id = "lipstick"
+	biological
+		hetcroL
+			name = "Heterochromia: Left"
+			id = "hetcroL"
+		hetcroR
+			name = "Heterochromia: Right"
+			id = "hetcroR"
+
+proc/select_custom_style(list/customization_types, mob/living/carbon/human/user as mob)
+	var/list/options = list()
+	for (var/styletype as anything in customization_types)
```

Comment:
would like these and similar to be typed variables

Reply:
Thank you, I've updated the code to use more explicit typing.